lib/internal/http2/core.js

```diff
@@ -160,6 +160,7 @@ const kLocalSettings = Symbol('local-settings');
 const kOptions = Symbol('options');
 const kOwner = owner_symbol;
 const kOrigin = Symbol('origin');
+const kPendingRequestCalls = Symbol('pending-request-calls');
```

Comment:
```suggestion
const kPendingRequestCalls = Symbol('kPendingRequestCalls');
```

(I know this is the current style in this file, but we [recommend matching descriptions](https://github.com/nodejs/node/blob/master/doc/guides/using-symbols.md) so that they can be more easily grep’ed for/looked up.)